docs/src/model_creation/network_analysis.md

```diff
@@ -500,6 +500,94 @@ Catalyst.reset_networkproperties!(rn)
+```
+Next, using `parameters` we note that an additional parameter, `Γ[1]` has been added to the system:
+```@example network_analysis_conservation_laws
+parameters(osys)
+```
+Here, Catalyst encodes all conserved quantities in a single, [vector-valued](@ref dsl_advanced_options_vector_variables), parameter called `Γ`.
+
+Practically, the `remove_conserved = true` argument can be provided when a `ReactionSystem` is converted to an `ODEProblem`:
+```@example network_analysis_conservation_laws
+using OrdinaryDiffEq, Plots
+u0 = [:X₁ => 80.0, :X₂ => 20.0]
+ps = [:k₁ => 10.0, :k₂ => 2.0]
+oprob = ODEProblem(rs, u0, (0.0, 1.0), ps; remove_conserved = true)
+nothing # hide
+```
+Here, while `Γ[1]` becomes a parameter of the new system, it has a [default value](@ref dsl_advanced_options_default_vals) equal to the corresponding conservation law. Hence, its value is computed from the initial condition `[:X₁ => 80.0, :X₂ => 20.0]`, and does not need to be provided in the parameter vector. Next, we can simulate and plot our model using normal syntax:
+```@example network_analysis_conservation_laws
+sol = solve(oprob)
+plot(sol)
+```
+!!! note
+    Any species eliminated using `remove_conserved = true` will not be plotted by default. However, it can be added to the plot using [the `idxs` plotting option](@ref simulation_plotting_options). E.g. here we would use `plot(sol; idxs = [:X₁, :X₂])` to plot both species.
+
+While `X₂` is an observable (and not unknown) of the ODE, we can [access it](@ref ref) just like if `remove_conserved = true` has not been used:
+```@example network_analysis_conservation_laws
+sol[:X₂]
+```
+!!! note
+    Generally, the `remove_conserved = true` should not change any model workflows. I.e. anything that works without this option should also work when an `ODEProblem` is created using `remove_conserved = true`.
+
+For any given `ReactionSystem` model, we can use the `conservationlaw_constants` to compute all a system's conserved quantities:
+```@example network_analysis_conservation_laws
+conservationlaw_constants(rs)
+```
+Next, the `conservedequations` function can be used to compute the algebraic equations produced when a system's conserved quantities are eliminated:
+```@example network_analysis_conservation_laws
+conservedequations(rs)
+```
+Finally, the `conservationlaws` functions yield a $m \times n$ matrix, where $n$ is a system's number of species, $m$ its number of conservation laws, and element $(i,j)$ correspond to the copy number of the $j$th species that occur in the $i$th conserved quantity:
+```@example network_analysis_conservation_laws
+conservationlaws(rs)
+```
+I.e. in this case we have a single conserved quantity, which contains a single copy each of the system's two species.
+
+!!! note
+    The `remove_conserved = true` option is available when creating `SDEProblem`s, `NonlinearProblem`s, and `SteadyStateProblem`s (and their corresponding systems). However, it cannot be used when creating `JumpProblem`s.
+
```

Comment:
I was thinking of adding a final example where I used a much less trivial model, just showing off what we can actually do.